src/domUpdates.js

```diff
@@ -35,20 +35,38 @@ let domUpdates = {
            <h2 class="recipe-name" tabindex="0">${recipe.name}</h2>
          </article>
        </article>`
-      })
+    })
 
   },
 
-  displayRecipeInfo(recipe, ingredients) {
+  displayRecipeInfo(recipe,cookables,missinIng) {
+    let cookable; 
+    
+    let isRecipeCookable = cookables.find(recipe =>{
+      return recipe.name === recipe
+    }) !== undefined 
+
+ 
+    if (isRecipeCookable) {
+      
+      cookable = 'You can cook this recipe!'
+    } else {
+      cookable = `Can't cook this recipe! <br>
+      You are missing: <br>${this.formatObjectsToDisplatCorrectly((this.figureMissing(recipe, missinIng)))}` 
+
+    }
+
     let trueInstructions = recipe.instructions.map(instruction => {
       return instruction.instruction;
-    }).join(',')
-
+    }).join('<br>')
+    recipeDisplay.innerHTML = ''
     recipeDisplay.innerHTML += 
     `<section class="recipe-display-main">
+    <h1>${cookable}</h1>
     <h1>${recipe.name}</h1>
     <div class="recipe-ingredients-with-cost">
-      <p class="popup-ingredients">${JSON.stringify(recipe.ingredients)}</p>
+    <h2>Nessasary Ingredient</h2>
```

Comment:
maybe this should be `Ingredients` on line 68 to account for multiple ingredients being shown (more readable)